composer.json

```diff
@@ -24,13 +24,15 @@
         "phpunit/phpunit": "~3.7|~4.0|~5.0",
         "symfony/finder": "~2.1|~3.0",
         "squizlabs/php_codesniffer": "~2.0",
-        "fabpot/php-cs-fixer": "~1.5"
+        "fabpot/php-cs-fixer": "~1.5",
+        "hoa/console": "~3.16|~1.14"
```

Comment:
This dependency is added for to support PHP 5.4 and 5.4.
https://travis-ci.org/bobthecow/psysh/builds/122592026